examples/credential_metadata_ldp_vc.json

```diff
@@ -16,35 +16,40 @@
         "type": [
           "VerifiableCredential",
           "UniversityDegreeCredential"
-        ],
-        "credentialSubject": {
-          "given_name": {
-            "display": [
-              {
-                "name": "Given Name",
-                "locale": "en-US"
-              }
-            ]
-          },
-          "family_name": {
-            "display": [
-              {
-                "name": "Surname",
-                "locale": "en-US"
-              }
-            ]
-          },
-          "degree": {},
-          "gpa": {
-            "mandatory": true,
-            "display": [
-              {
-                "name": "GPA"
-              }
-            ]
-          }
-        }
+        ]
       },
+      "claims": [
```

Comment:
Isn't `@context` required?

Reply:
for `ldp_vc`, it is required. https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#appendix-A.1.2.2-3.1.2.1

Reply:
```suggestion
      "@context": [
          "https://www.w3.org/2018/credentials/v1",
          "https://www.w3.org/2018/credentials/examples/v1"
        ],
        "claims": [
```